Kanchanop6509650229/CS360-QWERTYUIOP
# Ci docker

What does it do?
"Added Dockerfile for API and Client to support Docker-based deployment."

Why is it needed?
"Simplifies deployment steps by using Docker containers."
"Fixes the issue where the Content Manager did not display the correct publication status."

How to test it?
Run docker build -t my-app . to build the Docker image.
Run docker run -p 1337:1337 my-app to verify the API is running correctly.
Test file uploads in the Media Library and create a new Content-type.

Checklist:
- [x] สร้าง Docker image 
- [x] ทดสอบการรัน API และ Client
- [x] ตรวจสอบ Review Workflow
